extensions/arc/package.json

```diff
@@ -795,12 +835,12 @@
               ]
             },
             "requiredTools": [
-              {
-                "name": "azure-cli"
-              },
               {
                 "name": "azdata",
                 "version": "20.1.0"
+              },
+              {
+                "name": "kubectl"
```

Comment:
Nitpick: Can we keep the order consistent with sqlmi and datacontroller (kubectl first, then azdata)?